app/models/posts.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  after_create :posts_counter_update
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  def posts_counter_update
```

Comment:
- [ ] I wanted to make sure you are aware that when using the `after_create` option in your code, it is a requirement to place the method you are calling after the [private](https://guides.rubyonrails.org/active_record_callbacks.html#:~:text=before_validation%20%3Aensure_login_has_a_value%0A%0A%20%20private%0A%20%20%20%20def%20ensure_login_has_a_value%0A%20%20%20%20%20%20if%20login.nil%3F%0A%20%20%20%20%20%20%20%20self.login%20%3D%20email%20unless%20email.blank%3F%0A%20%20%20%20%20%20end%0A%20%20%20%20end) method.
This will help to maintain the structure and organization of your code.

🙂 Thanks for your understanding!

Reply:
👍